backend/benefit/applications/models.py

```diff
@@ -881,3 +882,34 @@ class ReviewState(models.Model):
 class AhjoSetting(TimeStampedModel):
     name = models.CharField(max_length=255, unique=True)
     data = JSONField()
+
+
+class AhjoStatus(TimeStampedModel):
+    """
+    Ahjo status of the application
+    """
+
+    status = models.CharField(
+        max_length=64,
+        verbose_name=_("status"),
+        choices=AhjoStatus.choices,
+        default=AhjoStatus.SUBMITTED_BUT_NOT_SENT_TO_AHJO,
+        db_index=True,
+    )
+    application = models.ForeignKey(
```

Comment:
Can application have multiple ahjo statuses? Have you considered `models.OneToOneField`?

Reply:
I guess  I had a very poor man's event sourcing type of idea going on here. So that all the statuses that the application goes through are on record.